conf/solr/8.11.1/solrconfig.xml

```diff
@@ -408,7 +408,7 @@
                       to occupy. Note that when this option is specified, the size
                       and initialSize parameters are ignored.
       -->
-    <filterCache class="solr.FastLRUCache"
+    <filterCache class="solr.search.CaffeineCache"

```

Comment:
My understanding is that we are changing this to prevent this error in server.log:

> Search Syntax Error: Error from server at http://localhost:8983/solr/collection1: Using join queries with synchronous filterCache is not supported! Details can be found in Solr Reference Guide under 'query-settings-in-solrconfig'.